src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```diff
@@ -60,14 +60,16 @@ public async Task StartAsync(CancellationToken cancellationToken)
             }
         }
 
-        public async Task StopAsync(CancellationToken cancellationToken)
+        public Task StopAsync(CancellationToken cancellationToken)
         {
-            _logger.LogDebug("Shuttingdown Rpc Channels Manager");
-            await _webHostRpcWorkerChannelManager.ShutdownChannelsAsync();
+            return Task.CompletedTask;
```

Comment:
Just curious, this method isn't really doing anything anymore - does it need to exist?